content/en/Product Updates/release-notes-november-2024.md

```diff
@@ -10,6 +10,25 @@ Explore What's New from Cobalt This Month
 {{% /pageinfo %}}
 
 
+## Enhanced AI/LLM Pentests Offering
+
+We have enhanced our AI/LLM offerings to provide customers with a more streamlined and intuitive experience:
+- The "Prompt Injection" methodology has been exclusively tailored as an agile test, offering a precise and focused scope representation.
+- We have rebranded "OWASP Top 10 LLMs" to "LLM" and positioned it as a comprehensive assessment, reflecting its full-spectrum coverage of LLM security.
+
+<b>Enhancements in Product:</b>
+- Catalog tile updated to reflect a starting credit price of 4 credits for agile tests.
+- Description of the "LLM" methodology updated to highlight its comprehensive nature.
+- Methodology selection streamlined: Agile method displays "Prompt Injection," while Comprehensive method displays "LLM."
+
+{{% pageinfo color="info" %}}
+For more information, have a look at our [AI/LLM Pentest Methodologies.](https://docs.cobalt.io/methodologies/ai-llm/)
```

Comment:
This link leads to a 404. Should it be https://docs.cobalt.io/getting-started/planning/#aillm-pentesting instead (cc @molfinn)?

Reply:
I believe it's because her PR isn't live yet, I went into her draft to pull the link that will be needed. However, if that page isn't going live for a bit, i'll need to pull the link off this draft.